src/network/networkd-address.c

```diff
@@ -437,6 +438,11 @@ int address_remove(
         assert(link->manager);
         assert(link->manager->rtnl);
 
+
+        r = in_addr_to_string(address->family, &address->in_addr, &b);
+        if (r == 0)
+                log_link_debug(link, "removing address %s", b);
+
```

Comment:
- It is not necessary to assign value to `r`.
- Please use `>= 0` instead of `== 0`.
- Please capitalize the message.
- This is just for debug logging. So, please use `DEBUG_LOGGING` macro.

So, I prefer something like following:
```
if (DEBUG_LOGGING) {
        _cleanup_free_ char *b = NULL;

        if (in_addr_to_string(...) >= 0)
                log_link_debug(link, "Removing address %s", b);
}
```

Reply:
thanks, wasn't ware of `DEBUG_LOGGING`

Reply:
Here, you can also drop `{}` after `if (DEBUG_LOGGING)` :-).